documentation/asset-tracker/topics/how-assettracker-works.md

```diff
@@ -0,0 +1,190 @@
+
+```{image} /asset-tracker/media/events.png
+:alt: AssetTracker events page
+:class: border-black
+```
+
+### Possible warnings
+
+- **Unknown Module:** Issued for the All location when AssetTracker detects a module that is not associated with a rack.
+- **Unknown Tag:** Issued at the rack level when a detected asset tag is not associated with a Hyperview asset.
+- **Collision:** Issued at the rack level when a detected tagged asset collides with another asset in the same position.
+- **Heartbeat Stopped:** Issued at the rack level when a module has not reported its status for more than 30 minutes.
+- **Module and Rack Size Mismatch:** Issued at the rack level when a module reports a size that does not match the rack it is associated with in Hyperview.
+
+### Viewing AssetTracker events
+
+1. Open the AssetTracker page (*Assets → AssetTracker*).
+2. Select the intended module → *Module Info*.
+3. Proceed to acknowledge or close event messages as appropriate. You can also navigate to the module's location using the View Location button.
+
+## Partial deployment
+
+The following table describes placement outcomes for an AssetTracker-licensed data center that has both tracked and untracked racks.
+
+:::{list-table}
+:header-rows: 1
+*  - Scenario
+   - Outcome
+*  - Asset tag is accidentally or intentionally removed
+   - The asset is moved to the current rack's Unplaced grid
+*  - Tagged asset is moved within a tracked rack
+   - The asset's RU is automatically updated in Hyperview
+*  - Tagged asset is moved to another tracked rack
+   - The asset's rack and RU is automatically updated in Hyperview
+*  - Tagged asset is moved to an untracked rack
+   - The asset is moved to the original rack's Unplaced grid, RU values must be updated manually
+*  - Untagged asset is moved to a tracked rack
+   - Undetected, RU values must be updated manually
+*  - Untagged asset is moved to an untracked rack
+   - Undetected, RU values must be updated manually
+:::
+
+## Known limitations
+
+AssetTracker has the following known limitations. We aim to address some of these in future releases.
+
+- AssetTracker can only place assets in the front of the rack.
+- Tagged zero U assets can be tracked to the specific rack, but not at the Rack Elevation level.
+- Shelved assets are not supported. AssetTracker will attempt to move tagged shelved assets to a non-shelf position in the Rack Elevation, or un-place them in the rack.
+- Side-by-side asset positions are currently not supported.
+
+## Tested barcode scanners
+
+:::{note}
+The following barcode scanners have been tested officially tested for AssetTracker. However, other scanners should work as well (provided they are properly configured; please refer to your scanner's documentation). The scanner must be connected to a device running Hyperview on a supported web browser to be able to input AssetTracker IDs.
```

Comment:
This sentence has "tested" twice:
"The following barcode scanners have been tested officially tested for AssetTracker."

Reply:
Fixed, Thanks!